Alexey/Text/count_of_words_in_a_string/tests/test_count_words_in_a_string.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+from Alexey.Text.count_of_words_in_a_string.main.count_words_in_a_string import words_counter
+
+
+class TestCountWordsInAString(unittest.TestCase):
```

Comment:
Please download txt file for 'War and Peas' by Tolstoy and enter it as a test value!

Reply:
576171